contrib/examples/walkthrough/README.md

```diff
@@ -1,4 +1,4 @@
 ## Walkthrough Resources
 
 This directory contains API resources for use with the [demo
-walkthrough](../../../docs/walkthrough.md).
+walkthrough](../../../docs/introduction.md).
```

Comment:
I think linking to the introduction doc here is a bit misleading; the new structure of the docs has the core of the walkthrough separated to its own doc(s), with the intro being much larger in scope. I would instead link to the 1.7 walkthrough doc.